pkg/metricscollector/v1beta1/tfevent-metricscollector/tfevent_loader.py

```diff
@@ -43,22 +44,22 @@ def find_all_files(directory):
 
     def parse_summary(self, tfefile):
         metric_logs = []
-        event_accumulator = EventAccumulator(tfefile, size_guidance={'tensors': 0})
+        event_accumulator = EventAccumulator(tfefile, size_guidance={TENSORS: 0})
         event_accumulator.Reload()
-        for tag in event_accumulator.Tags()['tensors']:
+        for tag in event_accumulator.Tags()[TENSORS]:
             for m in self.metric_names:
 
                 tfefile_parent_dir = os.path.dirname(m) if len(m.split("/")) >= 2 else os.path.dirname(tfefile)
                 basedir_name = os.path.dirname(tfefile)
                 if not tag.startswith(m.split("/")[-1]) or not basedir_name.endswith(tfefile_parent_dir):
                     continue
 
-                for wall_time, step, tensor in event_accumulator.Tensors(tag):
+                for tensor in event_accumulator.Tensors(tag):
                     ml = api_pb2.MetricLog(
-                        time_stamp=rfc3339.rfc3339(datetime.fromtimestamp(wall_time)),
+                        time_stamp=rfc3339.rfc3339(datetime.fromtimestamp(tensor.wall_time)),
                         metric=api_pb2.Metric(
                             name=m,
-                            value=str(tf.make_ndarray(tensor))
+                            value=str(tf.make_ndarray(tensor.tensor_proto))
```

Comment:
Since Tensorboard v2.12.0, the 'TensorEvent' typed was changed from the namedtuple to the dataclass.

ref: https://github.com/tensorflow/tensorboard/commit/1975529d953eff55d279ee036240e4db4cb0d57c

Also, I confirmed this script works under both `tensorboard==v2.11.0` and `tensorboard==v2.13.0`.